PactConsumerSwift.xcodeproj/project.pbxproj

```diff
@@ -729,7 +675,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi";
+			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint --strict\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```

Comment:
If we are adding `--strict` here, should we also do it in the build script? https://github.com/DiUS/pact-consumer-swift/blob/master/scripts/build.sh#L4

Do we even need it to be run in both places?

Reply:
Removed from this PR and will be addressed on a separate branch and PR.